google/oauth2/service_account.py

```diff
@@ -81,13 +83,20 @@
 from google.auth import metrics
 from google.oauth2 import _client
 
+_LOGGER = logging.getLogger(__name__)
+
 _DEFAULT_TOKEN_LIFETIME_SECS = 3600  # 1 hour in seconds
 _GOOGLE_OAUTH2_TOKEN_ENDPOINT = "https://oauth2.googleapis.com/token"
+_TRUST_BOUNDARY_LOOKUP_ENDPOINT = (
+    "iamcredentials.{}/v1/projects/-/serviceAccounts/{}/trustBoundary"
+)
+TRUST_BOUNDARY_LOOKUP_ENABLED_ENV = "TRUST_BOUNDARY_LOOKUP_ENABLED"
```

Comment:
I think "ENABLE_GOOGLE_TRUST_BOUNDARY" would be more clear and aligned with existing ENV variables.

Reply:
The case is we are going to support injection on this phase. And through the injection scenario. We never do the "look_up". When we onboard the "look_up", I think "LOOKUP_ENABLED" is more descriptive.